### DISCUSSION TITLE
Day-Time stamp for observations / events

### DISCUSSION THREAD

**Discussion by jibaer-izw** [olivierfriard/BORIS#695] [Ideas]
For our work we need a full timestamp (date + time) for the events of an observation. It would be nice if BORIS have an option to tag the videos with such a timestamp.

**Comment by olivierfriard** [comment]
A pre-release version of BORIS (8.22.10) is available that implements this feature:
https://github.com/olivierfriard/BORIS/releases/tag/v8.22.10

You have to set an offset (as date/time) when creating your observation.

Please test it and let me know.
Do not use it for production only for testing

**Reply by jibaer-izw** [reply]
Many thanks for you ultra fast response. I have just tested it, and it seems to work: I get valid timestamp for my events. This will be a great help for our work. Many thanks for this big piece of software.

More ideas for the export of tabular event data:
- To have an option to select the columns for export
- Additional export format Pandas/zipped Json (because everywhere are warnings about pickle files)
- An option for custom-exporter as python function

**Comment by olivierfriard** [comment]
Thanks for your suggestions
